### PR TITLE
Frontend

### PR DESCRIPTION
**Pull request by d-pilipavicius**
Frontend (basic business/product manipulation, DTO's) + Business Component finish up + User component extraction from the Business component. CHANGED HELPER NAME TO CommonHelper (PLEASE DO THAT TO EVERYTHING THAT IS NOT A COMPONENT, IT GETS CONFUSING TO MANAGE THE FOLDERS)

### CODE REVIEW

File: src/main/java/com/example/demo/CommonHelper/WebConfig.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.example.demo.CommonHelper;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
```
Comment (d-pilipavicius) [inline src/main/java/com/example/demo/CommonHelper/WebConfig.java:7]: Used for front-back communication allowance.

File: src/main/java/com/example/demo/CommonHelper/PostStartupConfig.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    userService.updateUser(newUser.getId(), UpdateUserDTO.builder()
+        .fullName(newUser.getFullName())
+        .businessId(newBusiness.getId())
+        .phoneNumber(newUser.getPhoneNumber())
+        .emailAddress(newUser.getEmailAddress())
+        .role(newUser.getRole())
+        .build());
+    System.out.println("User id: " + newUser.getId());
+    System.out.println("Business id: " + newBusiness.getId());
+  }
```
Comment (d-pilipavicius) [inline src/main/java/com/example/demo/CommonHelper/PostStartupConfig.java:47]: General method to populating the DB

**Review by xDziugas — APPROVED** [review]
viskas kaip ir +- chill, tikkad kai yra daug metodo parametru juos geriau visus kelt i naujas eilutes o ne kirst per puse nes matau kad visur pakeista

**Review by migle-kirilovaite — APPROVED** [review]
Seems good

**Pull request merged**